### PR TITLE
refactor (df/locale): system module

### PR DESCRIPTION
**Pull request by goel-aman**
## Description
Update dateformats and language with centralized data in system
module

## Related issues and discussion
#{Issue Number}

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] If you have multiple commits please combine them into one commit by squashing them.

- [ ] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by goel-aman** [comment]
Hey please review my pull request.

File: angular.json [hunk 1]
```diff
@@ -32,30 +32,36 @@
             ],
             "styles": [
               "src/main.scss",
-              {"inject":false,
+              {
```
Comment (Abhirup-99) [inline angular.json:35]: Remove changes to this file.

Comment (Abhirup-99) [reply angular.json:35]: The motive of your pr is not to edit these files. It would be better to have such changes in separate pr.

**Comment by Abhirup-99** [comment]
You still have those changes, try resetting head and then force push. Also, 1 commit per pr, so please update it.

**Comment by goel-aman** [comment]
@Abhirup-99  thanks for help buddy. I am new to open source thats why struggling a bit will get used to it soon.

**Review by Jov03 — APPROVED** [review]

**Pull request merged**